Lesson23-aysel/styles.css

```diff
@@ -0,0 +1,114 @@
+body {
+    display: grid;
+    place-items: center;
+    min-height: 100vh;
+}
+
+.background-gif {
+    width: 500px; 
+    height: 500px; 
+    background-image: url('https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/f/5f321b49-5fa7-407b-aaa0-cca6992d3d7c/d5s04qj-d26a072a-3294-4da7-8ab9-a1be78141275.gif?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1cm46YXBwOjdlMGQxODg5ODIyNjQzNzNhNWYwZDQxNWVhMGQyNmUwIiwiaXNzIjoidXJuOmFwcDo3ZTBkMTg4OTgyMjY0MzczYTVmMGQ0MTVlYTBkMjZlMCIsIm9iaiI6W1t7InBhdGgiOiJcL2ZcLzVmMzIxYjQ5LTVmYTctNDA3Yi1hYWEwLWNjYTY5OTJkM2Q3Y1wvZDVzMDRxai1kMjZhMDcyYS0zMjk0LTRkYTctOGFiOS1hMWJlNzgxNDEyNzUuZ2lmIn1dXSwiYXVkIjpbInVybjpzZXJ2aWNlOmZpbGUuZG93bmxvYWQiXX0.cBMxrJbp5Q5ghUGAev4ovPgb-Q2evrJs-XqbSrdUeB4');
+    background-size: cover; 
+    background-repeat: no-repeat;
+}
+
+.container {
+    display: flex;
+    align-items: center;
+    column-gap: 2rem;
+}
+
+img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.img {
```

Comment:
I also was a bit confused by this class since it's added to the div element, not img element. I'd name it something like 'img-container' instead so that it's clear that it's not for the image but for something else.

Reply:
In this way, when I make the img class as img-container, the images are nested and looks weird :/